ui/common/css/component/_continue-with.scss

```diff
@@ -1,6 +1,8 @@
 .continue-with {
   @extend %flex-column;
-
+  @media (hover: none) {
+    padding: 2em;
+  }
```

Comment:
What does padding have to do with the presence of a hovering device?

Reply:
In this PR, a larger 40x40 inline button style is used for all devices without a fine pointer. This is partly for simplicity, but also partly because iOS safari can't have overflow: visible on a fixed position parent with overflow: auto on a child and still handle scrolling properly.

The padding (for inline button style only) is meant to avoid the close button overlapping the corner of a "Play with" button on touch devices.